### PR TITLE
Add check for image age

### PR DESCRIPTION
**Pull request by martialblog**
Hi, 

I added a new check for the module:

- check_image_age will ensure that a containers image isn't older (days) than specified.

Since Docker images should regularly be rebuilt to ensure the latest (security) fixes, I think this will be a nice feature. 

Let me know if you require some changes to the PR, I'll take care of it.

Cheers, 
Markus

### CODE REVIEW

**Comment by martialblog** [comment]
Also, let me say: great plugin! The code and the tests were super easy to work with

**Comment by martialblog** [comment]
So the Python 3.6 and 3.7 Build are alright. However there are issues with the 3.4 and 3.5 Build unrelated to my changes.

3.5:
`Could not find a version that matches tox<3.7,==3.7.0,>=2.0,>=3.5 (from -r /tmp/pipenv-zc1pul9a-requirements/pipenv-mvdzdsug-constraints.txt (line 3))`

Updating the tox version might resolve that.

3.4: Has some issues with pytest_random_order... I'll try to find out what's going on here

**Comment by martialblog** [comment]
Seems as pytest-random-order==1.0.4 requires: Python >=3.5.0 

https://github.com/jbasko/pytest-random-order/releases/tag/v1.0.0

Cheers, 
Markus

**Comment by martialblog** [comment]
This might help: https://github.com/martialblog/check_docker/tree/fix-travis

**Comment by timdaman** [comment]
Very nice patch. Glad to hear the test worked for you. I will look into the test failures on Travis and Tox. Looks like you already made some progress on finding the solution, thanks.

**Comment by timdaman** [comment]
Can you rebase off my `freshen` branch. https://github.com/timdaman/check_docker/tree/freshen
I think that may resolve all your test failures. 🤞 If it works for you I will merge it into master so you can have a clean PR.

File: check_docker/check_docker.py [hunk 1]
```diff
@@ -799,6 +815,14 @@ def process_args(args):
                         metavar='WARN:CRIT',
                         help='Minimum container uptime in seconds. Use when infrequent crashes are tolerated.')
 
+    # Image Age
+    parser.add_argument('--image-age',
+                        dest='image_age',
+                        action='store',
+                        type=str,
+                        metavar='WARN:CRIT',
+                        help='Minimum image age in days.')
```
[the remark this answers is not in the document]
Comment (martialblog) [reply check_docker/check_docker.py]: Yes, I'll fix that

**Comment by martialblog** [comment]
Hi Tim, the Travis build works now. The codeclimate checks fail... but I think the checks need some configuration anyways. If you want I can have a look at that.

**Comment by timdaman** [comment]
Don't worry about CodeClimate. I cleared the alert.

**Pull request merged**